webhooks/virtualmachine/v1alpha2/validation/virtualmachine_validator_unit_test.go

```diff
@@ -260,6 +262,29 @@ func unitTestsValidateCreate() {
 			ctx.vm.Annotations[vmopv1.FirstBootDoneAnnotation] = updateSuffix
 		}
 
+		if args.extraLongNetworkIfName {
```

Comment:
I have a strong preference that we start to move away from this struct/table driven test because it is just getting too unwieldy and hard to follow. Later in this file there's a `Context("Network") { ...` where I added newer style tests for the Network stuff - let's move these test in there.

Reply:
gotcha. Done